python/lsst/obs/base/ingest_tests.py

```diff
@@ -162,35 +181,42 @@ def checkRepo(self, files=None):
         """
         pass
 
-    def _createRepo(self):
+    @classmethod
+    def _createRepo(cls):
         """Use the Click `testing` module to call the butler command line api
         to create a repository."""
         runner = LogCliRunner()
-        result = runner.invoke(butlerCli, ["create", self.root])
-        self.assertEqual(result.exit_code, 0, f"output: {result.output} exception: {result.exception}")
+        result = runner.invoke(butlerCli, ["create", cls.root])
+        assert result.exit_code == 0, f"output: {result.output} exception: {result.exception}"
```

Comment:
Why the change from assertEqual to a raw assert here and below?

Reply:
This is now a class method and `cls.assertEqual` does not work.

Reply:
Could you add a comment to that effect? It confused me when I saw it, given we don't use `assert` elsewhere.